astroplan/exceptions.py

```diff
@@ -0,0 +1,11 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+class AlwaysUpError(ValueError):
+    '''Target is circumpolar'''
```

Comment:
@bmorris3 Nitpick comment – always used triple double quotes for docstrings.

This is the Python coding style. It's mentioned in [pep8](https://www.python.org/dev/peps/pep-0008/#documentation-strings), but the command line `pep8` tool doesn't complain about triple single quotes as you use them everywhere.

There's [PEP257](https://www.python.org/dev/peps/pep-0257/) which says 
> For consistency, always use """triple double quotes""" around docstrings. Use r"""raw triple double quotes""" if you use any backslashes in your docstrings. For Unicode docstrings, use u"""Unicode triple-quoted strings""" .

IMO it's nice and important to have a coding style for Python, even if it seems pedantic sometimes.
It means that you don't have to learn a new one for every project you work on, and that devs don't have to argue.